### DISCUSSION TITLE
Tried Everything...new dexcom G7 sensor not connecting to Xdrip and don't have compatible Dexcom phone for G7 Dexcom app. Help please

### DISCUSSION THREAD

**Discussion by marcasroma** [NightscoutFoundation/xDrip#3832] [Bug reports]
I recently was upgraded from Dexcom G6 sensor to G7 only to discover that my phone and watch are not compatible....I can't buy a new phone right now so is there any way I can use Xdrip only or do I need another intermediary app? I have downloaded Xdrip app on phone but it won't connect to sensor when I put in transmitter ID. Any help would be greatly appreciated.

**Comment by Navid200** [comment]
What is the phone brand and model?  

If you follow this guide, you should be able to use only xDrip: https://navid200.github.io/xDrip/docs/Dexcom/G7.html 
But, you need to read it all and pay attention to every word.

**Reply by marcasroma** [reply]
Thanks! I managed to get it working. Now I have to get it on my Garmin watch! Thanks again.

**Reply by marcasroma** [reply]
Got it on my watch too now with a few more steps. All resolved. Thanks again for helping. Much appreciated.
